packages/openssl.rb

```diff
@@ -3,28 +3,11 @@
 class Openssl < Package
   description 'OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols.'
   homepage 'https://www.openssl.org/'
-  version '1.0.2o'
-  source_url 'https://github.com/openssl/openssl/archive/OpenSSL_1_0_2o.tar.gz'
-  source_sha256 'fd8bff81636c262ff82cb22286957d73213f899c1a80a3ec712c7ae80761ea9b'
+  version '1.0.2p-2'
```

Comment:
This can be `version '1.0.2p'` since we haven't updated in master yet.

Reply:
`skycocker:master` is already on 1.0.2p, my fork was just outdated.

Reply:
Right!  My bad.  Actually, there is a newer release available.  Might as well update?  https://github.com/openssl/openssl/archive/OpenSSL_1_0_2q.tar.gz

Reply:
Sure, sounds good!

Reply:
Sometimes I question how `make` works 😂
```
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Entering directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
make[2]: Leaving directory '/usr/local/tmp/crew/OpenSSL_1_0_2q.tar.gz.dir/openssl-OpenSSL_1_0_2q/engines'
```

Anyways, I'm currently testing the update, should be done fairly soon.

Reply:
If you want to make sure you have the latest changes in your fork, try this:
```
$ cd /path/to/chromebrew
$ git remote add chromebrew git@github.com:skycocker/chromebrew.git
$ git checkout master
$ git pull chromebrew master
$ git checkout fix-man
$ git merge master
```
You might have merge conflicts.  If you do, it might be easier just to delete and recreate the branch from a freshly updated master.

Reply:
Whoa! OpenSSL 1.0.2q has issues with parallel builds. We'd better save the update for another time.